integration_tests/bindc_03.py

```diff
@@ -0,0 +1,33 @@
+from ltypes import c_p_pointer, CPtr, pointer, i16, i32, Pointer, ccall, p_c_pointer
+
+@ccall
+def g(a: CPtr, value: i32) -> None:
+    pass
+
+@ccall
+def get_array(size: i32) -> CPtr:
+    pass
+
+@ccallable
```

Comment:
Since this is not called from C, it doesn't have to be `@ccallable` I think.

Reply:
Yes. I think I just took the example as it is from #609.

Reply:
Since its merged do you want `ccallable` to be removed in the next PR.